contrib/cray/Jenkinsfile.verbs

```diff
@@ -226,7 +212,7 @@ pipeline {
                 }
             }
         }
-        stage("Deploy: Stage 2") {
+        stage("Deploy") {
```

Comment:
Did you want to link to "stable" in the "Deploy" stage for "master" when it is successful?